### PR TITLE
Add V5+ and V5 nano documents

### PR DESCRIPTION
**Pull request by cuhome**
The V5+ and V5 nano are advanced autopilots designed in collaboration with the CUAV and PX4 teams and are based on the FMU V5 design.

### CODE REVIEW

File: en/SUMMARY.md [hunk 1]
```diff
@@ -16,6 +16,8 @@
 * [Basic Assembly](assembly/README.md)
   * [Mounting the Flight Controller](assembly/mount_and_orient_controller.md)
   * [Vibration Isolation](assembly/vibration_isolation.md)
+  * [V5+ Wiring Quickstart](assembly/quick_start_v5+.md)
```
Comment (hamishwillee) [inline en/SUMMARY.md]: Please prefix all of these in  SUMMARY with "CUAV" - ie "CUAV V5+ Wiring Quickstart", "CUAV V5+" etc.

File: en/assembly/quick_start_v5+.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
+# V5+ Wiring Quick Start
```
Comment (hamishwillee) [inline en/assembly/quick_start_v5+.md]: CUAV V5+ Wiring Quick Start

File: en/assembly/quick_start_v5+.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+---
```
Comment (hamishwillee) [inline en/assembly/quick_start_v5+.md]: Remove this horizontal rule - additional styling for headings not needed.

File: en/assembly/quick_start_v5_nano.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
+# V5+ Wiring Quick Start
+
+---
```
Comment (hamishwillee) [inline en/assembly/quick_start_v5_nano.md]: Remove this horizontal rule - additional styling for headings not needed.

File: en/assembly/quick_start_v5_nano.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
+# V5+ Wiring Quick Start
```
Comment (hamishwillee) [inline en/assembly/quick_start_v5_nano.md]: Should be "nano". So 

> CUAV V5 Nano Wiring Quick Start

File: en/assembly/quick_start_v5_nano.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+
+## Telemetry System (Optional) 
+
+A telemetry system allows you to communicate with, monitor, and control a vehicle in flight from a ground station (for example, you can direct the UAV to a particular position, or upload a new mission).
+
+The communication channel is via Telemetry Radios. The vehicle-based radio should be connected to the **TELEM1/TELEM2** port (if connected to this port, no further configuration is required). The other radio is connected to your ground station computer or mobile device (usually via USB).
+
+![quickstart](../../assets/flight_controller/v5_nano/connection/v5_nano_quickstart_07.png)
+
+## SD Car(Optional)
```
Comment (hamishwillee) [inline en/assembly/quick_start_v5_nano.md]: Typo - missing "d" in card.

File: en/flight_controller/v5+.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
+# V5+ AutoPilot
```
Comment (hamishwillee) [inline en/flight_controller/v5+.md]: Add CUAV whenever you refer to this board in a heading please - so here: CUAV V5+ Autopilot

File: en/flight_controller/v5+.md [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+Any multicopter / airplane / rover or boat that can be controlled with normal RC servos or Futaba S-Bus servos. The complete set of supported configurations can be seen in the [Airframes Reference](../airframes/airframe_reference.md).
+
+## Further info
+
+- [FMUv5 reference design pinout](https://docs.google.com/spreadsheets/d/1-n0__BYDedQrc_2NHqBenG1DNepAgnHpSGglke-QQwY/edit#gid=912976165). 
+- [CUAV Github](https://github.com/cuav) 
+- [Base board design reference](https://github.com/cuav/hardware/tree/master/V5_Autopilot/V5%2B/V5%2B_BASE)
+- [V5+ Manual](http://manual.cuav.net/V5-Plus.pdf)
+
+
```
Comment (hamishwillee) [inline en/flight_controller/v5+.md]: Please delete all the empty space.

File: en/SUMMARY.md (hunk 1 above)
Comment (cuhome) [reply en/SUMMARY.md]: The flight controller is named v5+. Do you think I need to add a manufacturer's trademark?

**Comment by hamishwillee** [comment]
There are a lot of minor issues with this. I'll push a PR to your branch rather than identifying every issue.

**Comment by hamishwillee** [comment]
Closing in favour of #521

**Comment by cuhome** [comment]
图片

**Pull request closed without merging**